test/unittests/commonMocks.h

```diff
@@ -36,133 +40,163 @@
 
 /* ****************************************************************************
 *
-* USING 
+* USING
 */
 using ::testing::_;
```

Comment:
Oh yes, that is the intention, no using in header files, only `using xx::yy` in cpp files.
Must have missed this one.

Unfortunately, the script doesn't flag this as error, so it is done manually.

Good catch! Must be fixed!